rpc/handler.go

```diff
@@ -327,8 +327,22 @@ func (h *handler) handleCall(cp *callProc, msg *jsonrpcMessage) *jsonrpcMessage
 	if err != nil {
 		return msg.errorResponse(&invalidParamsError{err.Error()})
 	}
-
-	return h.runMethod(cp.ctx, msg, callb, args)
+	start := time.Now()
+	answer := h.runMethod(cp.ctx, msg, callb, args)
+
+	// Collect the statistics for RPC calls if metrics is enabled.
+	// We only care about pure rpc call. Filter out subscription.
```

Comment:
Seems like you filter out `unsubscribe`, but not `subscribe`. Is that intended? 
Otherwise, you can define a flag `noMetering` and set to `true` in the checks above regarding subscriptions.

Reply:
The `subscribe` is handled by `handleSubscribe`.